src/style.css

```diff
@@ -1 +1,105 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+  border: 0;
+
+  font-weight: 300;
+}
```

Comment:
Avoid using the universal selector (*) as it impacts performance. It's better to set box-sizing to border-box specifically for the elements that require it.